CMakeLists.txt

```diff
@@ -404,6 +416,14 @@ else()
   if(NOT TARGET fmt::fmt)
     message(FATAL_ERROR "fmt not found! Have you disabled the git submodules (BOUT_UPDATE_GIT_SUBMODULE)?")
   endif()
+  # Build the library in <build dir>/lib: this makes updating the path
+  # for bout-config much easier
+  set_target_properties(fmt PROPERTIES
+    LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/lib"
+    ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/lib")
+  set(FMT_INCLUDE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/externalpackages/fmt/include")
+  set(CONFIG_CFLAGS "${CONFIG_CFLAGS} -I\${FMT_INCLUDE_PATH}")
+  set(CONFIG_LDFLAGS "${CONFIG_LDFLAGS} -lfmtd")
```

Comment:
Hmm, it's definitely `libfmtd` on my machine, I'm not sure why that is. Could be because I'm using a debug build? Will need to look into this further.

Reply:
I configured with `cmake .. -DCMAKE_INSTALL_PREFIX=$(pwd)/../installed -DBOUT_USE_PETSC=ON` and ended up with `libfmt.a`.
Fedora also ships `libfmt.so.7.0.3` ...

Reply:
Ah, they set `FMT_DEBUG_POSTFIX` to `d` which gets used if `CMAKE_BUILD_TYPE=Debug`. I think this is a Windows thing (debug builds are often not ABI compatible with non-debug builds).

We can override that, so I'll push a fix shortly